### PR TITLE
fix : use event bus from @grafana/data, remove SystemJS runtime

### PR DESCRIPTION
**Pull request by muradsater**
**Problem / User story**
Update implementation after grafana review

### CODE REVIEW

File: src/appEventHandler.ts [hunk 1]
```diff
@@ -1,14 +1,12 @@
-import { AppEvent } from '@grafana/data';
-import { SystemJS } from '@grafana/runtime';
+import { AppEvent, EventBusSrv } from '@grafana/data';
 import { getCalculationWarnings, getLimitsWarnings, stringifyError } from './cdf/client';
 import { failedResponseEvent, responseWarningEvent } from './constants';
 import { SuccessResponse } from './types';
 
-export const appEventsLoader = SystemJS.load('app/core/app_events');
+export const bus = new EventBusSrv();
```
Comment (abdullah-cognite) [inline src/appEventHandler.ts:6]: Renaming this to `busService` or `eventBusService` would` be a good idea, just `bus` feels a bit ambiguous

**Comment by abdullah-cognite** [comment]
Apart from the small comment above, the PR looks good to me 🚀

**Comment by polomani** [comment]
Seems to work, let's do the refactor suggested by @abdullah-cognite and merge

**Comment by muradsater** [comment]
> Seems to work, let's do the refactor suggested by @abdullah-cognite and merge

Could you apply the change @abdullah-cognite ? Zsolt is not available at the moment

**Review by abdullah-cognite — APPROVED** [review]

**Pull request merged**